framework/src/fviks/FVInterfaceKernel.C

```diff
@@ -227,3 +227,24 @@ FVInterfaceKernel::neighborFromFace(const bool correct_skewness) const
           correct_skewness,
           _face_info->neighborPtr()->subdomain_id()};
 }
+
+Moose::SingleSidedFaceArg
+FVInterfaceKernel::singleSidedFaceArg(const MooseVariableFV<Real> & variable,
```

Comment:
This is not currently covered in framework testing. It would be nice to have it covered, but I won't say it's a requirement to get this merged.

Reply:
Whoever creates the next interface kernel will catch it. My attempts at creating a one-way flow constraint were unsuccessful (would not converge) but that utility was useful so i figure let's keep it